reference/ClosureBlog.md

```diff
@@ -0,0 +1,65 @@
+The magic of simple, serverless, Functions.
+
+Pulumi offers a flexible and simple approach to create powerful 'Serverless Functions' starting from no more than just a simple JavaScript function.  That function has very few limitations on it, and allows one to think of your 'Serverless Function' code as being a direct part of your application, instead of a separate infrastructure artifact that needs to be managed, versioned, and processed independently.
+
+Many Cloud providers, like AWS, Azure and GCP, have all embraced the idea of providing [Function as a Service](https://en.wikipedia.org/wiki/Function_as_a_service) (FaaS) functionality to their platforms.  This functionality allows users of that platform to easily run logic for their cloud applications, without necessarily having to build or otherwise maintain the necessary underlying infrastructure for that logic themselves.  FaaS systems normally differ from more established [Platform as a Service](https://en.wikipedia.org/wiki/PaaS) (PaaS) systems in that they tend to be able to 'spin down' to using no cloud compute resources when not in use, while also being able to 'spin up' to respond to requests very quickly (ideally in milliseconds).  These systems are commonly called 'serverless' which, while being a complete misnomer (of course servers are involved!), does help convey the idea that in general you are free to squint and to not think about those servers *too much*.
```

Comment:
Probably this whole para can just be dropped. I don't mind it as it's SEO juice, but means we have a lot of words up front. The other 3 paras are good philosophy though and I'd want to preserve most of that.